.github/workflows/ci.yml

```diff
@@ -14,10 +14,10 @@ jobs:
       AUTH0_API_AUDIENCE: ${{ secrets.AUTH0_API_AUDIENCE }}
     strategy:
       matrix:
-        node: [14, 16, 18, 20]
```

Comment:
let's add 22

Reply:
@simoneb Is this a blocking request? Would you mind if I add this on a fast follow-up? The PR author is OOO.